src/pages/p2p/index.tsx

```diff
@@ -66,10 +65,8 @@ const derivP2PPortalData = {
 
 const DP2PHome = () => {
     const [is_mounted] = usePageLoaded() // needed to fix the second Hero-component during page's loadin
-    const { is_p2p_allowed_country } = useRegion()
-    const { website_status } = useWebsiteStatus()
-    const clients_country = website_status?.clients_country
-    if (clients_country !== undefined) {
+    const { is_p2p_allowed_country, is_p2p_loading } = useRegion()
+    if (!is_p2p_loading) {
         if (is_p2p_allowed_country) {
```

Comment:
```suggestion
        if (!is_p2p_loading && is_p2p_allowed_country) {
```

Reply:
They both have different condition. Such as, if is_p2p_loading is true then it will show loading, and after that if is_p2p_allowed_country is it will show p2p page otherwise it 404 page will appear.

Reply:
Thank you @hasan-deriv 🙏